apps/heft/src/utilities/resolvePackage.ts

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { sync } from 'resolve';
+import * as path from 'path';
+import { FileSystem, PackageJsonLookup } from '@rushstack/node-core-library';
+
+const packageJsonLookup: PackageJsonLookup = new PackageJsonLookup();
+
+/**
+ * Options for package path resolution
+ * @public
+ */
+export interface IResolvePackageOptions {
+  doNotResolveSymlinks: boolean;
+}
+
+/**
+ * Gets the path of a given package name
+ * @public
+ */
+export function resolvePackage(
```

Comment:
This is exported, but nothing seems to use it.  Also, it's marked as `@public` but not actually exported (nor should it be, since functions do not make good public APIs).